openedx/adg/lms/tests/test_helpers.py

```diff
@@ -0,0 +1,26 @@
+""" Tests for LMS helper """
+
+import pytest
+
+from common.djangoapps.student.tests.factories import UserFactory, UserProfileFactory
+from openedx.adg.lms.helpers import get_user_first_name
+
+from .constants import FIRST_NAME, FIRST_PART_OF_FULL_NAME
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'first_name, full_name, expected_first_name', [
+        (FIRST_NAME, '', FIRST_NAME),
+        ('', '{} helper'.format(FIRST_PART_OF_FULL_NAME), FIRST_PART_OF_FULL_NAME)
```

Comment:
The string '{} helper' is a bit confusing. Maybe, for more clarity, we can refactor it to:

`'{first_name} {last_name}'.format(first_name=FIRST_NAME, last_name=LAST_NAME)
`
Plus, I don't see the need for two different variables `FIRST_NAME` and `FIRST_PART_OF_FULL_NAME`, I think we can just use `FIRST_NAME` for both tests.

Reply:
Yes `'{} helper'` is confusing, but I kept them separate to add some readable distention b/w both of the test cases, otherwise in both cases the expected results would be the FIRST_NAME. So adding FIRST_PART_OF_FULL_NAME is making this case distinct and readable? But we minimizing the confusion I've changed it to `f'{FIRST_PART_OF_FULL_NAME} last_part_of_full_name'` which reads much better. WDYT now?